src/Neuron/Zettelkasten/View.hs

```diff
@@ -105,7 +105,11 @@ renderZettel (store, graph) zid = do
         div_ [class_ "column"] $ do
           div_ [class_ "ui header"] "Navigate up"
           let forestB = obviateRootUnlessForest zid $ dfsForestBackwards zid graph
-          ul_ $ renderForest Nothing (LinkTheme_Simple $ Just zid) store graph forestB
+          ul_ $ do
+            li_ $ do
+              a_ [class_ "zettel-link item", href_ "/z-index.html", title_ "z-index"] $ do
+                span_ [class_ "zettel-link-title"] "z-index"
```

Comment:
Respecting the [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), I'd extract [this code](https://github.com/srid/neuron/blob/44365c6/src/Neuron/Zettelkasten/Link/View.hs#L61-L66) into a function, and then re-use it here.

Reply:
Extracted the two links, active and regular. Although active is not a link is used in place of one.

They aren't exactly the same to justify using the same function I think.